README.md

```diff
@@ -29,7 +29,7 @@ You will make your changes in this new file.
 more information about password hash : https://linuxconfig.org/how-to-hash-passwords-on-linux
 
 **keyboard Layout:**
-* HOST: update the list of classes in srv_fai_config/class/99-seapath to remove the FRENCH class if you prefer an english keyboard layout ;)
+* HOST: create a grub item without the FRENCH flag (see "Classes Customization")
```

Comment:
I don't understand this sentence. Will the host class create a new grub entry ? Or do we talk about the default grub entry with us keyboard ?

Reply:
in this commit, FRENCH is still the default, and if you want to change that, you just have to remove the "french" flag in grub, and so you do this by creating a specific grub item (with the whiptail interface).
Don't worry too much about this since it will evolve with a upcoming commit setting US as the default and requiring the user to specifically add the FRENCH or GERMAN class to change this.